src/nowplaying.html

```diff
@@ -15,35 +15,35 @@ <h2 class="nowPlayingPageTitle"></h2>
                         <div class="nowPlayingArtist nowPlayingSerie"></div>
                     </div>
                     <div class="nowPlayingPageUserDataButtonsTitle"></div>
-                    
+
                 </div>
-                
+
                 <div class="sliderContainer flex">
                     <div class="positionTime"></div>
                         <div class="nowPlayingPositionSliderContainer">
                             <input type="range" is="emby-slider" pin step="1" min="0" max="100" value="0" class="nowPlayingPositionSlider" data-slider-keep-progress="true" />
                         </div>
                     <div class="runtime"></div>
                 </div>
-                
+
                 <div class="nowPlayingButtonsContainer focuscontainer-x justify-content-space-between">
-                
+
                     <div class="nowPlayingInfoButtons">
 
                         <button is="paper-icon-button-light" class="btnCommand btnRepeat repeatToggleButton autoSize" title="${ButtonRepeat}"
                             data-command="SetRepeatMode">
                             <span class="material-icons repeat"></span>
                         </button>
-                    
+
                         <button is="paper-icon-button-light" class="btnRewind btnNowPlayingRewind btnPlayStateCommand autoSize" title="${Rewind}">
                             <span class="material-icons replay_10"></span>
                         </button>
-                        
+
                         <button is="paper-icon-button-light" class="btnPreviousTrack btnPlayStateCommand autoSize" title="${ButtonPreviousTrack}">
                             <span class="material-icons skip_previous"></span>
                         </button>
 
-                        <button is="paper-icon-button-light" class="btnPlayPause btnPlayStateCommand autoSize" title="${ButtonPause}">
+                        <button is="paper-icon-button-light" class="btnPlayPause btnPlayStateCommand autoSize cardOverlayButtonIcon" title="${ButtonPause}">
```

Comment:
Now `Play/Pause` is not lined up.
Before
![play-old](https://user-images.githubusercontent.com/56478732/87243555-97623180-c43f-11ea-89f5-50858e8c0346.png)
After
![play-new](https://user-images.githubusercontent.com/56478732/87243558-992bf500-c43f-11ea-9dac-733d2fa6d65d.png)

Reply:
It looks even to me when testing locally, do you have a fix I could apply for your screen?

Reply:
I have removed `cardOverlayButtonIcon` and got `Before` one.
Desktop layout also has uneven `Play/Pause` button. But it is bigger without `cardOverlayButtonIcon`.
Ah, this is Chrome vs Firefox issue. Chrome sucks here.